public/singleRest.js

```diff
@@ -1,74 +0,0 @@
-//fetching restaurant names to links in nav bar
-let navbar = document.getElementById("nav-links");
-fetch("./api/restaurants")
-  .then((res) => {
-    return res.json();
-  })
-  .then((allNames) => {
-    allNames.forEach((restaurant) => {
-      let resid = restaurant.id;
-      let resname = restaurant.name;
-      console.log(resname);
-      let anchor = document.createElement("a");
-      let listName = document.createElement("li");
-      anchor.href = "/" + resid;
-      anchor.onclick = listName.textContent = restaurant.name;
-      navbar.appendChild(anchor);
-      anchor.appendChild(listName);
-    });
-  });
```

Comment:
Reference this fetch if you wanted to programmatically drop your markers and popups on the yelpington-index.js